EIPS/eip-2926.md

```diff
@@ -92,7 +103,7 @@ Note: there could be an edge case when computing FIO for the chunks where the da
 
 ### Gas cost of code-accessing opcodes
 
-Details of how the code is stored, is left to the client implementers. However, to reflect the removal of the max code limit and the fact that larger codes will be more expensive to load, a uniform read access cost of 200 warming read cost and an extra 500 write costs are being charged.
+Details of how the code is stored, is left to the client implementers. However, to reflect the removal of the max code limit and the fact that larger codes will be more expensive to load, reading a non-accessed chunk will incur a 200 warming cost, and chunks written beyond the 24kb limit will cost 500 gas instead of 200.
```

Comment:
I just realized that this might be a big problem, especially in the sense of `EXTCODECOPY`. I assume that we also charge these gas costs. We will need to analyze if this breaks anything, because this will now massively increase the cost of copying large amounts (for instance copying all the code of a contract). 

Reading after 24 KiB limit does this incur a **read** cost of 500 gas? (It now says "written" which might be confused with code deposit?)

Reply:
We also need to specify how we track the accessed code chunks. It is clear from a transaction which has non-reverting/aborting call frames. But what if a call frame aborts? Is the code accessed in that frame then still considered "accessed"? (I would assume: yes, but then we have to specify this :smile: :+1: )